runtime/doc/options.txt

```diff
@@ -3390,6 +3390,27 @@ A jump table for the options with a short description can be found at |Q_op|.
 	buffers, for which window and buffer related autocommands can be
 	ignored indefinitely without affecting the global 'eventignore'.
 
+	Note: The following events cannot be ignored by 'eventignorewin' and
```

Comment:
Might be good to mention the reasoning behind it. E.g. something like "The following events are considered to happen outside of window context and thus cannot be ignored by 'eventignorewin'.